VERSION "Khalil Light Control System 1.0"

NS_ :
    NS_DESC_
    CM_
    BA_DEF_
    BA_
    VAL_
    CAT_DEF_
    CAT_
    FILTER
    BA_DEF_DEF_
    EV_DATA_
    ENVVAR_DATA_
    SGTYPE_
    SGTYPE_VAL_
    BA_DEF_SGTYPE_
    BA_SGTYPE_
    SIG_TYPE_REF_
    VAL_TABLE_
    SIG_GROUP_
    SIG_VALTYPE_
    SIGTYPE_VALTYPE_
    BO_TX_BU_
    BA_DEF_REL_
    BA_REL_
    BA_DEF_DEF_REL_
    BU_SG_REL_
    BU_EV_REL_
    BU_BO_REL_
    SG_MUL_VAL_

BS_:

BU_: MASTER SLAVE

BO_ 257 LOW_BEAM_CTRL: 2 MASTER
 SG_ LOW_BEAM_STATUS : 0|8@1+ (1,0) [0|1] "" SLAVE
 SG_ LOW_BEAM_MODE : 8|8@1+ (1,0) [0|3] "" SLAVE

BO_ 258 HIGH_BEAM_CTRL: 2 MASTER
 SG_ HIGH_BEAM_STATUS : 0|8@1+ (1,0) [0|1] "" SLAVE
 SG_ HIGH_BEAM_MODE : 8|8@1+ (1,0) [0|3] "" SLAVE

BO_ 259 PARKING_LEFT_CTRL: 2 MASTER
 SG_ PARKING_LEFT_STATUS : 0|8@1+ (1,0) [0|1] "" SLAVE
 SG_ PARKING_LEFT_MODE : 8|8@1+ (1,0) [0|3] "" SLAVE

BO_ 260 PARKING_RIGHT_CTRL: 2 MASTER
 SG_ PARKING_RIGHT_STATUS : 0|8@1+ (1,0) [0|1] "" SLAVE
 SG_ PARKING_RIGHT_MODE : 8|8@1+ (1,0) [0|3] "" SLAVE

BO_ 261 HAZARD_LIGHTS_CTRL: 2 MASTER
 SG_ HAZARD_LIGHTS_STATUS : 0|8@1+ (1,0) [0|1] "" SLAVE
 SG_ HAZARD_LIGHTS_MODE : 8|8@1+ (1,0) [0|3] "" SLAVE

BO_ 262 RIGHT_TURN_CTRL: 2 MASTER
 SG_ RIGHT_TURN_STATUS : 0|8@1+ (1,0) [0|1] "" SLAVE
 SG_ RIGHT_TURN_MODE : 8|8@1+ (1,0) [0|3] "" SLAVE

BO_ 263 LEFT_TURN_CTRL: 2 MASTER
 SG_ LEFT_TURN_STATUS : 0|8@1+ (1,0) [0|1] "" SLAVE
 SG_ LEFT_TURN_MODE : 8|8@1+ (1,0) [0|3] "" SLAVE

VAL_ 257 LOW_BEAM_STATUS 0 "DEACTIVATED" 1 "ACTIVATED" 255 "FAILED" 254 "INVALID";
VAL_ 257 LOW_BEAM_MODE 0 "FAHREN" 1 "STAND" 2 "PARKING" 3 "WOHNEN";

VAL_ 258 HIGH_BEAM_STATUS 0 "DEACTIVATED" 1 "ACTIVATED" 255 "FAILED" 254 "INVALID";
VAL_ 258 HIGH_BEAM_MODE 0 "FAHREN" 1 "STAND" 2 "PARKING" 3 "WOHNEN";

VAL_ 259 PARKING_LEFT_STATUS 0 "DEACTIVATED" 1 "ACTIVATED" 255 "FAILED" 254 "INVALID";
VAL_ 259 PARKING_LEFT_MODE 0 "FAHREN" 1 "STAND" 2 "PARKING" 3 "WOHNEN";

VAL_ 260 PARKING_RIGHT_STATUS 0 "DEACTIVATED" 1 "ACTIVATED" 255 "FAILED" 254 "INVALID";
VAL_ 260 PARKING_RIGHT_MODE 0 "FAHREN" 1 "STAND" 2 "PARKING" 3 "WOHNEN";

VAL_ 261 HAZARD_LIGHTS_STATUS 0 "DEACTIVATED" 1 "ACTIVATED" 255 "FAILED" 254 "INVALID";
VAL_ 261 HAZARD_LIGHTS_MODE 0 "FAHREN" 1 "STAND" 2 "PARKING" 3 "WOHNEN";

VAL_ 262 RIGHT_TURN_STATUS 0 "DEACTIVATED" 1 "ACTIVATED" 255 "FAILED" 254 "INVALID";
VAL_ 262 RIGHT_TURN_MODE 0 "FAHREN" 1 "STAND" 2 "PARKING" 3 "WOHNEN";

VAL_ 263 LEFT_TURN_STATUS 0 "DEACTIVATED" 1 "ACTIVATED" 255 "FAILED" 254 "INVALID";
VAL_ 263 LEFT_TURN_MODE 0 "FAHREN" 1 "STAND" 2 "PARKING" 3 "WOHNEN";
